guide/additional-info/async-await.md

```diff
@@ -109,24 +109,17 @@ But since all of these react methods are started at the same time, it would just
 ```js
 client.on('message', message => {
 	if (message.content === `${prefix}react`) {
-		message.react('🇦').then(() => {
-			message.react('🇧').then(() => {
-				message.react('🇨').catch(error => {
-					// handle failure of the third react
-				});
+		message.react('🇦')            
+			.then(() => message.react('🇧'))
+			.then(() => message.react('🇨'))
+			.catch(err => {
+				// handle failure of any Promise rejection inside here
 			})
-			.catch(error => {
-				// handle failure of the second react
-			});
-		})
-		.catch(error => {
-			// handle failure of the first react
-		});
 	}
 });
 ```
 
-This code looks really messy and has redundant catch methods, so let's look how the same code would look with async/await.
+This is an special form of resolving mutiple Promises because we use the `.then()` callback to chain resolving Promises with each other and if one of the Promises gets rejected the `.catch()` does fire. So let's look how the same code would look with async/await.
```

Comment:
1. I dunno how "special" promise chains are.
2. Technically the `catch` callback is called, not fired.
3. Might be useful to link to documentation on promise chains (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then#Chaining maybe?)